app/assets/javascripts/discourse/components/user-selector.js.es6

```diff
@@ -94,11 +94,20 @@ export default TextField.extend({
             }
             return i.username ? i.username : i;
           });
+
+          let previouslySelected = [];
```

Comment:
can you add a comment explaining what it's doing here ? it's not very clear at first sight.

user-selector will be replaced at some point and it would help ensure wee keep consistent behavior.

Reply:
That makes sense, let me add it 👍